src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -52,9 +58,34 @@
   export default {
     name: 'LicenseIconography',
     components: {
-      FontAwesomeIcon
+      FontAwesomeIcon,
+      Popup
     },
     props: {
+      showPopup: {
+        type: Boolean,
+        default: false
+      },
+      popupDirection: {
+        type: String,
+        default: 'right'
+      },
+      popupAction: {
+        type: String,
+        default: 'hover'
+      },
+      popupDelay: {
+        type: Number,
+        default: 0
+      },
+      popupColor: {
+        type: String,
+        default: 'black'
+      },
```

Comment:
These props will automatically bind to `Popup`.

Reply:
Sorry, I did not get what you are suggesting. Can you be more elaborate?
Thank you.

Reply:
Since the props `direction`, `action` and `delay` are not used by `LicenseIconography` but rather passed to `Popup`, there is no need for them to be defined on the component. The line `v-bind="$attrs"` on `Popup` will automatically capture any props leftover by `LicenseIconography`. 

So you could write 
```
<LicenseIconography direction="left" delay="1000"/>
```
and the props `direction` and `delay` would be sent to the inner `Popup` component.

Reply:
More information about `inheritAttrs` and `v-bind="$attrs"` can be found [here](https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance) and a blog post on the same [here](https://blog.usejournal.com/the-perfect-wrapper-components-in-vue-2-6-and-soon-vue-3-0-41520cd4c3f2).